scripts/prune_lm_dir.py

```diff
@@ -425,17 +425,37 @@ def IterateOnce(threshold, step, iter, recovery_step, recovery_size):
                 break
 
     """
-    global logprob_changes, steps, current_num_ngrams
+    global logprob_changes, logprob_changes_for_KL, steps, current_num_ngrams
```

Comment:
it's not clear to me what this KL is.  In any case there shouldn't be caps in these variable names.  can you try to explain it somewhere?
Perhaps here by the initialization of the variables.